web/src/stream_edit.js

```diff
@@ -534,6 +558,12 @@ export function initialize() {
         });
     });
 
+    $("#streams_overlay_container").on(
+        "click",
+        ".stream-notification-container .sub_setting_control",
+        stream_notification_reset,
+    );
```

Comment:
Why does this selector target the name `sub_setting_control`? It seems very non-specific as a class name for this button.